### PR TITLE
FF7: resume any music after battle in WM

### PR DESCRIPTION
**Pull request by myst6re**

### CODE REVIEW

File: src/music.cpp [hunk 1]
```diff
@@ -531,17 +472,34 @@ uint32_t ff7_music_sound_operation_fix(uint32_t type, uint32_t param1, uint32_t
...
 	if (*unk1)
 	{
-		ff7_next_battle_world_id = *unk3;
+		ff7_next_battle_world_id = *battle_id;
 
 		// Now we know the battle scene ID, so we can try to customize battle music in Worldmap too
-		stop_music();
-		ff7_play_midi(7);
+		// Play music
+		((uint32_t(*)(uint32_t, uint32_t, uint32_t, uint32_t, uint32_t, uint32_t))ff7_externals.sound_operation)(
```
Comment (myst6re) [inline src/music.cpp:500]: Oops, removing stop_music() is a bad idea, at least the current music id must be discarded

Comment (julianxhokaxhiu) [reply src/music.cpp:500]: Yes, that's why I used it, in order to flush the current id actually.

Comment (myst6re) [reply src/music.cpp:501]: Done

**Comment by myst6re** [comment]
Should fix #202

**Review by julianxhokaxhiu — APPROVED** [review]

**Comment by julianxhokaxhiu** [comment]
Thank you!

**Pull request merged**